src/Plugin/CookiePlugin.php

```diff
@@ -167,13 +167,13 @@ private function createCookie(RequestInterface $request, string $setCookieHeader
      *
      * @param string $part A single cookie value in format key=value
      *
-     * @return string[]
+     * @return array{0:string, 1:?string}
      */
     private function createValueKey(string $part): array
     {
         $parts = explode('=', $part, 2);
         $key = trim($parts[0]);
-        $value = isset($parts[1]) ? trim($parts[1]) : true;
+        $value = isset($parts[1]) ? trim($parts[1]) : null;
```

Comment:
Not a BC break: the Cookie class only accepts string or null as a value. This is a bug fix.